{"uuid": "3dea752e-65aa-41b3-bab1-da973a73905e", "children": ["8e2672dd-62c6-4db0-875a-e7ef553e7992"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=96.0.4664.110)\n\n", "trace": "  File \"C:\\Selenium\\Selenium-Python-Framework\\venv\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Selenium\\Selenium-Python-Framework\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1117, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Selenium\\Selenium-Python-Framework\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 915, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Selenium\\Selenium-Python-Framework\\tests\\conftest.py\", line 40, in setup\n    actions.sendKeys(lp.getUsername(), \"justin.sirunwa@nihilent.com\")\n  File \"C:\\Selenium\\Selenium-Python-Framework\\pageObjects\\LoginPage.py\", line 15, in getUsername\n    return self.driver.find_element(*LoginPage.username)\n  File \"C:\\Selenium\\Selenium-Python-Framework\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 976, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"C:\\Selenium\\Selenium-Python-Framework\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Selenium\\Selenium-Python-Framework\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1639669020071, "stop": 1639669024266}], "start": 1639669020071, "stop": 1639669024565}